osu.Game/Screens/Play/Player.cs

```diff
@@ -355,10 +363,15 @@ private Drawable createOverlayComponents(WorkingBeatmap working)
                         Anchor = Anchor.Centre,
                         Origin = Anchor.Centre
                     },
-                    skipOverlay = new SkipOverlay(DrawableRuleset.GameplayStartTime)
+                    skipIntroOverlay = new SkipOverlay(DrawableRuleset.GameplayStartTime)
                     {
                         RequestSkip = performUserRequestedSkip
                     },
+                    skipOutroOverlay = new SkipOverlay(Beatmap.Value.Storyboard.LatestEventTime ?? 0)
+                    {
+                        RequestSkip = () => updateCompletionState(true),
+                        Alpha = 0
+                    },
```

Comment:
While I think this is fine for now for simplicity, there may be a point in the future where this needs to be refactored into a single overlay able to handle multiple skip operations (for instance, when implementing #12493).

Reply:
Yeah, not to mention that the current way would break when rewinding before the storyboard outro part, which can be fixed easily with a single skip overlay (potentially by using the existing `PeriodTracker` to tell whether a period can be skipped).

Reply:
Rather than a reply to an inline comment, that seems like something you should post in a review? It is an oversight that blocks this from being usable.

Reply:
Right, I thought it's not easily solvable and something to resolve in a follow-up PR or something, but now that I think about it, it can be solved and does currently block it from being usable, will do.